packages/twenty-server/src/database/commands/upgrade-version/0-23/0-23-update-activities.command.ts

```diff
@@ -214,11 +228,25 @@ export class UpdateActivitiesCommand extends CommandRunner {
 
           await timelineActivityRepository.update(
             {
+              name: 'task.created',
+              linkedObjectMetadataId: activityObjectMetadataId,
+              linkedRecordId: activity.id,
+            },
+            {
+              linkedObjectMetadataId: taskObjectMetadataId,
+              name: 'linked-task-created',
```

Comment:
I thought it was linked-task.created? linked-note.updated, etc.

Reply:
Forgot to push fix...